views/xibo-ssp-connector-form-javascript.twig

```diff
@@ -69,6 +69,9 @@
 
     dialog.closest('.modal').addClass('modal-big');
 
+    // Get the filter value
+    let filter = $('select[name="campaignFilter"]').val();
```

Comment:
Is the field `'select[name="campaignFilter"]'` inside of the `dialog`? If so, maybe it's better to use the query on that dialog instead with `dialog.find('select[name="campaignFilter"]')`?

Reply:
Thanks for this! Could you please take a look at this again based on this [commit](https://github.com/xibosignage/xibo-cms/pull/2598/commits/02e8d1a991179bed685ec48c20388056ec7d3b2e)?